### DISCUSSION TITLE
I want to add phantom wallet in modal

### DISCUSSION THREAD

**Discussion by FaizanIbrahimSL** [WalletConnect/walletconnect-monorepo#4428] [Developer Support]
HI,
Can anyone please help me to solve my issue i am struggling with this last 7 days.

I am using "@walletconnect/modal-react-native" and want to add phantom wallet init as well but i am unable to add phantom wallet
Here is my code:

import "@walletconnect/react-native-compat";
import { WalletConnectModal } from "@walletconnect/modal-react-native";

const providerMetadata = {
  name: "droppPhygital",
  description: "droppPhygital description",
  url: "https://droppthinklab.xyz/",
  icons: ["https://droppthinklab.xyz/favicon.ico"],
  redirect: {
    native: "droppPhygital://",
    universal: "droppthinklab.xyz",
  },
};

const sessionParams = {
  namespaces: {
    eip155: {
      methods: ["eth_sendTransaction", "personal_sign"],
      chains: ["eip155:1"],
      events: ["chainChanged", "accountsChanged"],
      rpcMap: {},
    },
    solana: {
      methods: ["eth_sendTransaction", "personal_sign"],
      chains: ["solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp"],
      events: ["chainChanged", "accountsChanged"],
      rpcMap: {},
    },
  },
};

const App = () => {
  return (
      <GestureHandlerRootView style={{ flex: 1 }}>
        <Provider store={store}>
          <PersistGate loading={null} persistor={persistor}>
            <StatusBar backgroundColor={Color.black} barStyle="light-content" />
            {/* <ErrorBoundary> */}
            <WalletConnectModal
              sessionParams={sessionParams}
              projectId={stagProjectId}
              providerMetadata={providerMetadata}
              explorerExcludedWalletIds={"ALL"}
              explorerRecommendedWalletIds={[
                "c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96",
                "ecc4036f814562b41a5268adc86270fba1365471402006302e70169465b7ac18",
                "a797aa35c0fadbfc1a53e7f675162ed5226968b44a19ee3d24385c64d1d3c393",
                "fd20dc426fb37566d803205b19bbc1d4096b248ac04548e3cfb6b3a38bd033aa",
              ]}
            />
            <MainNavigator />
            <Toast config={toastConfig} />
            {/* </ErrorBoundary> */}

            {/* <NetworkLogger /> */}
          </PersistGate>
        </Provider>
      </GestureHandlerRootView>
    
  );
};

**Comment by Cali93** [comment]
Hey @FaizanIbrahimSL,

Unfortunately Phantom wallet doesn't support the WalletConnect protocol.
